src/platform/web.rs

```diff
@@ -200,3 +202,23 @@ pub enum PollStrategy {
     #[default]
     Scheduler,
 }
+
+pub trait CustomCursorExtWebSys {
+    /// Creates a new cursor from a URL pointing to an image.
+    /// It uses the [url css function](https://developer.mozilla.org/en-US/docs/Web/CSS/url),
+    /// so the same image formats and paths are supported.
```

Comment:
While we are at it, this might be a bit misleading, even though the specification mandates this, no browser actually supports all image formats they should be supporting.

Again, this is actually a bug, but maybe we could write something like this:
```suggestion
    /// It uses the [url css function](https://developer.mozilla.org/en-US/docs/Web/CSS/url),
    /// but browser support for image formats is inconsistent. Using [PNG] is recommended.
    ///
    /// [PNG]: https://en.wikipedia.org/wiki/PNG
```

Reply:
That seems nicer.